File: pkg/java/src/main/java/dev/openfga/language/validation/ValidationOptions.java
package dev.openfga.language.validation;

import dev.openfga.language.validation.Validator.Rules;

public class ValidationOptions {

    private String typePattern = String.format("^%s$", Rules.TYPE);
    private String relationPattern = String.format("^%s$", Rules.RELATION);

    public String getTypePattern() {
        return typePattern;
    }

    public void setTypePattern(String typePattern) {
        this.typePattern = typePattern;
    }

    public String getRelationPattern() {
        return relationPattern;
    }

    public void setRelationPattern(String relationPattern) {
        this.relationPattern = relationPattern;
    }
}

File: pkg/csharp/src/OpenFga.Language/Validation/ValidationOptions.cs
namespace OpenFga.Language.Validation;

public class ValidationOptions
{
    private string _typePattern = $"^{Validator.Rules.Type}$";
    private string _relationPattern = $"^{Validator.Rules.Relation}$";

    public string TypePattern
    {
        get => _typePattern;
        set => _typePattern = value;
    }

    public string RelationPattern
    {
        get => _relationPattern;
        set => _relationPattern = value;
    }
}
